### ISSUE TITLE
[TW-44] Space handling in dateformat

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#106]
_Louis-Claude Canon on 2012-04-21T15:27:32Z says:_ 

 Using a space in dateformat prevents from putting due date easily:

<pre>
$ task rc.dateformat:"d/m/Y H:N" add Test due:21/4/2012 15:00
Configuration override rc.dateformat:d/m/Y H:N
Found extra operands.
</pre>

However, it works fine without the space:

<pre>
$ task rc.dateformat:"d/m/YH:N" add Test due:21/4/201215:00
Created task 1.
Configuration override rc.dateformat:d/m/YH:N
</pre>

Obtained with the package task_2.0.0-0ubuntu2~oneiric2_i386.deb on Ubuntu 11.10.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2012-04-21T15:27:32Z
Modified: 2014-07-02T00:13:24Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-04-21T15:39:08Z says:_ 

 Merci Louis-Claude.

This is likely just the shell separating "due:21/4/2012" from "15:00".  Could you please try quoting the 'due' value the same way you quote the dateformat:

<pre>
task rc.dateformat:"d/m/Y H:N" add Test due:"21/4/2012 15:00"
</pre>

**Comment by taskwarrior** [comment]
_Louis-Claude Canon on 2012-04-21T15:44:33Z says:_ 

 You are welcome. Thank you for this nice tool, I am really enjoying it.

The result is the same (with bash 4.2.10) :

<pre>
$ task rc.dateformat:"d/m/Y H:N" add Test due:"21/4/2012 15:00"
Configuration override rc.dateformat:d/m/Y H:N
Found extra operands.
</pre>

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-04-21T17:05:53Z says:_ 

 Here is a workaround, until 2.0.1 is released:

<pre>
task rc.dateformat:"d/m/Y H:N" add Test due:\"21/4/2012 15:00\"
</pre>

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2012-07-08T19:08:54Z says:_ 

 Deferring to 2.2.0 because this needs new argument binary classifiers, which are not nearly ready yet.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-02-11T04:36:20Z says:_ 

 Deferring again - this is one for the upgraded parser.

**Comment by taskwarrior** [comment]
_Benjamin Weber on 2014-05-26T20:13:45Z says:_ 

 Fix Version/s: 2.4.0

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-07-02T00:13:24Z says:_ 

 Fixed in conjunction with failing #628 unit test.
